File: FirstYearLab/Lab Sessions 2/Session 11 (B)/q2.py
print("Yogesh Kumar Bhatt")
print("Enter the numbers: ")
lst=[]
prime=[]
while(1):
    a=input()
    if a=="END":
        break
    lst.append(int(a))
for i in range(len(lst)):
    if lst[i]>1:
        for j in range(2,lst[i]):
            if(lst[i]%j==0):
                break
        else:
            prime.append(lst[i])
print("So the prime numbers are: ",prime)

File: FirstYearLab/Lab Sessions 2/Session 9/q2.c
#include <stdio.h>

int main()
{
    printf("Yogesh Kumar Bhatt");
    int arr[50], i;

    FILE *fptr = fopen("Numbers.txt", "r");
    for (i = 0; i <= 25; i++)
        fscanf(fptr, "%d", &arr[i]);

    printf("\n");
    for (i = 0; i <= 25; i++)
        if (arr[i] % 7 == 0)
            printf("%d ", arr[i]);
    
    return 0;
}